spec/components/discourse_spec.rb

```diff
@@ -78,22 +78,28 @@ def enabled?
     let(:plugin1) { plugin_class.new.tap { |p| p.enabled = true; p.path = "my-plugin-1" } }
     let(:plugin2) { plugin_class.new.tap { |p| p.enabled = false; p.path = "my-plugin-1" } }
 
-    before { Discourse.plugins.append(plugin1, plugin2) }
-    after { Discourse.plugins.clear }
+    before do
+      Discourse.plugins.append(plugin1, plugin2)
+    end
+
+    after do
+      Discourse.plugins.delete plugin1
+      Discourse.plugins.delete plugin2
+    end
 
     before do
       plugin_class.any_instance.stubs(:css_asset_exists?).returns(true)
       plugin_class.any_instance.stubs(:js_asset_exists?).returns(true)
     end
 
     it 'can find plugins correctly' do
-      expect(Discourse.plugins).to contain_exactly(plugin1, plugin2)
+      expect(Discourse.plugins).to include(plugin1, plugin2)
```

Comment:
Doesn't this change the behaviour of the assertion? `contain_exactly` asserts that the array contains only the two plugins while the latter asserts that the array includes the given plugins but does not ensure it does not contain other plugins.

Reply:
Got it :+1: